helm/flowforge/templates/deployment.yaml

```
@@ -198,6 +200,9 @@ spec:
       {{- end }}
       {{- end }}
       volumes:
+      - name: tempdir
+        emptyDir: {}
+        sizeLimit: 1Gi      
```

```suggestion
        emptyDir: 
          sizeLimit: 1Gi      
```

This didn't validate, looking at the spec it should be under emptyDir

---

https://kubernetes.io/docs/concepts/storage/volumes/#emptydir-configuration-example